src/Resources/contao/library/Contao/Controller.php

```diff
@@ -1003,6 +1003,7 @@ public static function redirect($strLocation, $intStatus=303)
 			exit;
 		}
 
+		$strLocation = ltrim($strLocation, '/');
```

Comment:
Why would you want to detect wrong input. The method just doesn't work if you pass a leading `/` so why do you think it's wrong if it ensures that there's none automatically? And yes, we could move it down there if you prefer.